src/components/CreationForm.vue

```diff
@@ -71,30 +71,6 @@
 							:style="{ backgroundColor: newBoard[fieldId] }" />
 					</NcColorPicker>
 				</div>
-				<NcMultiselect v-else-if="field.type === 'select'"
-					:value="newBoard[fieldId]"
-					:options="Object.values(field.options)"
-					label="label"
-					:placeholder="field.placeholder"
-					@input="setSelectValue(fieldId, $event)"
-					@search-change="query = $event">
-					<template #option="{option}">
-						<component :is="option.icon"
-							v-if="option.icon"
-							class="option-icon"
-							:size="20" />
-						<NcHighlight :text="option.label" :search="query" class="option-title multiselect-option-title" />
-					</template>
-					<template #singleLabel="{option}">
-						<component :is="option.icon"
-							v-if="option.icon"
-							class="multiselect-label-icon"
-							:size="20" />
-						<span class="option-title">
-							{{ option.label }}
-						</span>
-					</template>
-				</NcMultiselect>
```

Comment:
Why is this replaced without replacement?

Reply:
Because it is not used.